### ISSUE TITLE
Issue compressing normal maps

### ISSUE THREAD

**Issue by PeterHiber** [BinomialLLC/basis_universal#102]
Hi there, thanks for the awesome library! We are having some issues compressing normal maps, here's an example:

Input normal map
![original](https://user-images.githubusercontent.com/54840231/72422578-060d4d80-3783-11ea-875f-abbb145756b0.png)

Compressed normal map
![example_unpacked_rgb_BC7_M5_RGBA_0_0000](https://user-images.githubusercontent.com/54840231/72422605-0efe1f00-3783-11ea-95cf-a67625ff7c59.png)

We tried some different parameters but could not manage to make the results significantly better. Then we found the following snippet in the readme:

"For tangent space normal maps, you should separate X into RGB and Y into Alpha, and provide the compressor with 32-bit/pixel input images. Or use the "-separate_rg_to_color_alpha" command line option which does this for you. The internal texture format that Basis Universal uses (ETC1S) doesn't handle tangent space normal maps encoded into RGB well. You need to separate the channels and recover Z in the pixel shader using z=sqrt(1-x^2-y^2)."

For reasons beyond our control this is likely not something we are capable of doing in the near future. But we thought it might still be worthwhile to ask and get confirmation.

1. Is it possible to improve the quality of the compressed normal map without splitting and shader reconstruction (i.e., are we missing some parameter or using the library the wrong way)?
2. Or do we have to adapt our pipeline to split and reconstruct (if using basis for normal maps)?

We are thankful for answers, have a pleasant day. :)

## Reproducing:

Using latest basisu from git (895ee8ee7e, 2020-01-04), built with latest Visual Studio 2019 (16.4.3), release build.

Compress the original normal map using:

`basisu original.png -q 128 -normal_map -comp_level 5 -mipmap -mip_renorm`

Then decompress using:

`basisu original.basis`

**Comment by slimbuck** [comment]
Hi @PeterHiber,

We recently went through the process of adding basis texture support to Playcanvas and also encountered issues with normal map quality (see https://github.com/BinomialLLC/basis_universal/issues/99).

I would suggest using "-q 255" instead of 128. This should make the biggest difference to your textures quality-wise.

Ultimately we found that the only way of getting good quality normal maps is compressing to swizzled GGGR format, as the docs suggest.
